sdk/tables/azure-data-tables/azure/data/tables/__init__.py

```diff
@@ -16,15 +16,14 @@
     Metrics,
     RetentionPolicy, TableAnalyticsLogging, TableSasPermissions, CorsRule, UpdateMode, SASProtocol, Table,
 )
-from ._shared.models import (
+from ._models import (
     LocationMode,
     ResourceTypes,
     AccountSasPermissions,
-    TableErrorCode
 )
-from ._shared.policies import ExponentialRetry, LinearRetry
+from ._policies import ExponentialRetry, LinearRetry
 from ._version import VERSION
-
+from ._deserialize import TableErrorCode
```

Comment:
nit: Is `TableErrorCode` something that users will use? I'm going with yes, since it's in this init file. Based on that, I think it might be better to have it in the _models file, the general rule of thumb I operate under is that file contains the user exposed models

Reply:
I agree, however this creates a circular dependency because `process_table_error` and `deserialize_table_creation` use TableErrorCode. in `_models.py` there's three imports from `_deserialize.py`. Any advice on how to structure those imports/functions to move `TableErrorCode` around?

Reply:
Ah I see, I think it's ok to leave there then